MyBlog.API/Controllers/UserController.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System.Net.Mime;
+using Microsoft.AspNetCore.Mvc;
+using MyBlog.Infrastructure.Entities;
+using MyBlog.Service.Areas.Users;
+
+namespace MyBlog.Controllers;
+
+[ApiController]
+[Route("api/users")]
+[Produces(MediaTypeNames.Application.Json)]
+public class UserTestController : ControllerBase
+{
+    private readonly ILogger<UserTestController> _logger;
+    private readonly IUserService _userService;
+    
+    public UserTestController(ILogger<UserTestController> logger, IUserService userService)
+    {
+        _logger = logger;
+        _userService = userService;
+    }
+
+    /// <summary>
+    /// Get list of Users
+    /// </summary>
+    /// <returns>Returns User list</returns>
+    /// <response code="200">Success</response>
+    [HttpGet]
+    [ProducesResponseType(typeof(List<User>), StatusCodes.Status200OK)]
+    public async Task<IActionResult> GetListAsync()
+    {
+        var users = await _userService.GetListAsync();
+        
+        return Ok(users);
+    }
+
+    /// <summary>
+    /// Get User by id
+    /// </summary>
+    /// <param name="id">User id</param>
+    /// <returns>Returns User</returns>
+    /// <response code="200">Success</response>
+    /// <response code="404">UserNotFound</response>
+    [HttpGet("{id:int}")]
+    [ProducesResponseType(typeof(User), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    public async Task<IActionResult> GetByIdAsync([FromRoute] int id)
+    {
+        var user = await _userService.GetByIdAsync(id);
+    
+        return Ok(user);
+    }
+    
+    /// <summary>
+    /// Creates a User.
+    /// </summary>
+    /// <param name="user">User object</param>
+    /// <returns>Created User object</returns>
+    /// <response code="200">Success</response>
+    [HttpPost]
+    [ProducesResponseType(typeof(int), StatusCodes.Status200OK)]
+    public async Task<IActionResult> CreateAsync([FromBody] User user)
+    {
+        var userId = await _userService.CreateAsync(user);
+
+        var userFromBd = await _userService.GetByIdAsync(userId);
+        
```

Comment:
probably, you wanna name variable: User From Database, so it will be userFromDb, not userFromBd
but i recomend you to name this just user
fix it in all your project